### ISSUE TITLE
[CRAN] dbConnect disregards `embedded = dbdir`

### ISSUE THREAD

**Issue by cronacronis** [hannesmuehleisen/MonetDBLite#75]
Looks like `MonetDBLite` disregard `embedded = dbdir` parameter and creates db in `/tmp/Rtmp8FqQpF`.

```
> library(data.table)
> dt_big <- data.table(columna = c("a"))
> library(MonetDBLite)
> library(DBI)
> dbdir = "/home/cracs/repos/git/analysisengine/monetdblite"
> con <- dbConnect(MonetDBLite::MonetDBLite(), embedded = dbdir)
> dbWriteTable(con, "dt_big", dt_big)
[1] TRUE
> ms <- MonetDBLite::src_monetdblite(dbdir = dbdir)
Error in monetdb_embedded_startup(embedded, !getOption("monetdb.debug.embedded",  : 
  MonetDBLite cannot change database directories (already started in /tmp/Rtmp8FqQpF, shutdown first).
```

**Comment by hannesmuehleisen** [comment]
does it work if you use

     con <- dbConnect(MonetDBLite::MonetDBLite(), dbdir)

**Comment by cronacronis** [comment]
Yes it does.

**Comment by ajdamico** [comment]
@hannesmuehleisen this is a bug.  here's the problem:  when the folder in embedded=dbdir does not exist on the local machine, dbConnect() decides that the directory was actually not specified and simply uses `tempdir()` instead.  then src_monetdblite tries to open it up in the correct directory and it fails, as it should.

i think dbConnect() should either create the non-existent directory with a warning, or it should fail and force the user to create the directory first?


	library(DBI)
	dbdir = tempdir()
	con <- dbConnect(MonetDBLite::MonetDBLite(), embedded = dbdir)
	ms <- MonetDBLite::src_monetdblite(dbdir = dbdir)
	# works fine obviously


	# # # separate console

	library(DBI)
	dbdir = paste0( tempdir() , "directory_that_does_not_exist" )
	con <- dbConnect(MonetDBLite::MonetDBLite(), embedded = dbdir)
	# here is the clue--  dbConnect did not use dbdir--
	dbGetInfo(con)$gdk_dbpath
	# [1] "C:\\Users\\anthonyd\\AppData\\Local\\Temp\\RtmpagBz91"

	# it used tempdir() instead
	tempdir()
	# [1] "C:\\Users\\anthonyd\\AppData\\Local\\Temp\\RtmpagBz91"

	# so now this breaks
	ms <- MonetDBLite::src_monetdblite(dbdir = dbdir)
	# Error in monetdb_embedded_startup(embedded, !getOption("monetdb.debug.embedded",  : 
	  # MonetDBLite cannot change database directories (already started in C:\Users\anthonyd\AppData\Local\Temp\RtmpagBz91, shutdown first).

**Comment by vadimnazarov** [comment]
Hi, I have the same problem on OS X Mavericks on both console R and RStudio.
R version: 3.2.3
RStudio: 0.99.878

```
dbdir = tempdir()
con <- dbConnect(MonetDB.R::MonetDB(), embedded = dbdir)
Error: 'monetdb_embedded_startup' is not an exported object from 'namespace:MonetDBLite'
```

```
dbdir = tempdir()
con <- dbConnect(MonetDBLite::MonetDBLite(), embedded = dbdir)
Error: 'monetdb_embedded_startup' is not an exported object from 'namespace:MonetDBLite'
```

Also it doesn't work if I supply already existing directory, same error. It seems that it worked for me sometimes ago at this summer, but I don't remember at what time I updated / changed something.

**Comment by seaneuron** [comment]
Same problem as above. I use Win 8.1

R version 3.3.1 (2016-06-21)
RStudio Version 0.99.903

`dbdir = tempdir()
con <- dbConnect(MonetDB.R::MonetDB(), embedded = dbdir)
Error: 'monetdb_embedded_startup' is not an exported object from 'namespace:MonetDBLite'`

**Comment by ajdamico** [comment]
@vadimnazarov @seaneuron don't you want `MonetDBLite::MonetDBLite()`?

**Comment by vadimnazarov** [comment]
I'm not sure what happened, but I restarted RStudio and R and now `MonetDBLite::MonetDBLite()` works ok for me in both cases.

@ajdamico It didn't work, I tried both of them, but right now it works despite that I didn't change anything except reloading R / Rstudio. Probably it was the same problem as above with directories with started MonetDB process in them. I think I may mess up something like starting two MonetDB in parallel in the same tempdir.
